packages/@react-aria/focus/src/FocusRing.tsx

```diff
@@ -32,7 +32,9 @@ interface FocusRingProps {
   /** Whether the element is a text input. */
   isTextInput?: boolean,
   /** Whether the element will be auto focused. */
-  autoFocus?: boolean
+  autoFocus?: boolean,
+  /** Whether the element we're applying focus ring to is disabled. */
+  isDisabled?: boolean
```

Comment:
Did you figure out *why* `useFocus` is not firing an `onBlur` already when the button is disabled? We aren't passing `isDisabled` through to it so it should.

Reply:
ach, i forgot to look more into that, lemme get back to you